exp/controllers/machinepool_controller.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Similar to the api types, I'm wondering if it would be good to have the controller for each "experiment" (or logical groups of experiments) in it's own package, that would limit the amount of potential for accidental code sharing between experiments and would make migration out of experimental easier.